simphony_ui/liggghts_model.py

```diff
@@ -14,48 +15,184 @@ class LiggghtsModel(HasStrictTraits):
     #: The input file used for liggghts.
     input_file = File()
 
-    boundary_condition_x = Enum('periodic', 'fixed', 'shrink-wrapped')
-    boundary_condition_y = Enum('periodic', 'fixed', 'shrink-wrapped')
-    boundary_condition_z = Enum('periodic', 'fixed', 'shrink-wrapped')
+    # Boundary conditions parameters
+    x_wall = Enum('periodic', 'fixed')
```

Comment:
ok, you mostly already have the extended name around, and we have tab completion, so I'd say x_wall_boundary_condition. Shortening is always a double edge knife, because you have to do additional mapping of what that abbreviation means, which might not be immediate.